draft-jaju-httpbis-zstd-window-size.md

```diff
@@ -50,33 +50,38 @@ informative:
 --- abstract
 
 Deployments of Zstandard, or "zstd", can use different window sizes to limit
-memory usage during compression and decompression. Some browsers and user agents
-limit window sizes to mitigate memory usage concerns, causing interoperability
-issues. This document updates the window size limit in RFC8878 from a
-recommendation to a requirement in HTTP contexts.
+memory usage during compression and decompression. Some browsers and user
+agents limit window sizes to mitigate memory usage concerns, causing
+interoperability issues. This document updates the window size limit in RFC8878
+from a recommendation to a requirement in HTTP contexts.
 
 
 --- middle
 
 # Introduction
 
 Zstandard, or "zstd", specified in {{?RFC8878}}, is a lossless data compression
-mechanism similar to gzip. When used with HTTP, the "zstd" Content Encoding
+mechanism similar to gzip. When used with HTTP, the "zstd" Content Coding
 token signals to the decoder that the content is Zstandard-compressed.
 
-Deployments of Zstandard can use different window sizes to configure the maximum
-memory a decoder requires to decompress a frame. Larger window sizes tend to
-improve the compression ratio, but cause more memory to be used. {{RFC8878}}
-provides a recommendation for decoders to support window sizes up to 8 MB, and
-for encoders to not generate frames requiring window sizes larger than 8 MB.
-However, it is just a recommendation ({{!RFC8878, Section 3.1.1.1.2}}).
+An important property of Zstandard-compressed content is its Window_Size
+({{!RFC8878, Section 3.1.1.1.2}}), which describes the maximum distance for
+matches and therefore how much of the content must be kept in memory during
+decompression. Larger Window_Size values tend to improve the compression ratio,
```

Comment:
```suggestion
Deployments of Zstandard use different Window_Size values
({{!RFC8878, Section 3.1.1.1.2}}) to describe the maximum back-reference distance,
which controls the amount of content that must be kept in memory during
decompression. Larger Window_Size values tend to improve the compression ratio,
```
Proposed a diff that combines the two approaches with somewhat cleaner phrasing.
People can probably guess at the meaning of "matches", but maybe "back-references" is better since it's also used in RFC8878?